engine/src/main/java/org/terasology/rendering/dag/nodes/OutputToScreenNode.java

```diff
@@ -53,8 +53,8 @@ public OutputToScreenNode(Context context) {
         displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
         worldRenderer = context.get(WorldRenderer.class);
 
-        WorldRenderer worldRenderer = context.get(WorldRenderer.class);
-        requiresCondition(() -> worldRenderer.getCurrentRenderStage() == MONO || worldRenderer.getCurrentRenderStage() == LEFT_EYE);
+        WorldRenderer renderer = context.get(WorldRenderer.class);
+        requiresCondition(() -> renderer.getCurrentRenderStage() == MONO || renderer.getCurrentRenderStage() == LEFT_EYE);
```

Comment:
Is this generating a warning? There are a number of renderers in Terasology and unless strictly necessary I'd like to keep this variable as `worldRenderer`, for the time being.

Reply:
It warns here about an [HiddenField] because we already have an `worldRenderer` at class level. At a second look we could use the `worldRenderer` from the class here since line 54 and 56 are identical.
Also this warning seems to be triggered very often in the rendering code (e.g. WorldRendererImpl).

Reply:
I have no idea how that managed to slip under the radar. There has been plenty of work done on the renderer in recent time, but mostly only by two people. I guess such a small number of eyeballs will let some issues go through. 

Line 56 is obviously redundant. Please revert to the previous version of the code and then remove that line: that's the error to correct.

Reply:
For only two people it is quite hard to catch everything. Even my two extra eyes needed a second look to catch the error.